lib/xslt/iso2mods.xsl

```diff
@@ -927,7 +927,7 @@
                 </dc:format>
                 <!-- Output Dataset# point, linestring, polygon, raster, etc. -->
                 <dc:type>
-                  <xsl:text>Dataset#</xsl:text>
+                  <xsl:text>Dataset#Raster</xsl:text>
```

Comment:
How do we know that the type will always be Raster? Can't it be `Dataset#Polygon`, etc. for vector data?

Reply:
Good question, perhaps we need some conditional in the XSL ... like on `$geometryType` ?